cmd/gen-go-sample/main.go

```diff
@@ -353,14 +353,38 @@ func (g *generator) genSample(sampConf schema_v1p2.Sample, methConf GAPICMethod)
 
 	p := g.pt.Printf
 
-	// Region tag, function signature and new client
+	// Region tag
 	argStr, err := argListStr(initInfo, g)
 	if err != nil {
 		return err
 	}
 	p("")
 	p("// [START %s]", sampConf.RegionTag)
 	p("")
+
+  // comments
+  emptyArr := make([]string, 0)
+  requiresNewLine := false
+  
+  if sampConf.Description != "" {
+  	writeComment(sampConf.Description, emptyArr, g)
+  	requiresNewLine = true
+  }
+
+  for i, argName := range initInfo.argNames {
+  	comment := initInfo.argTrees[i].comment
+  	if comment == "" {
+  		continue
+  	}
+  	if requiresNewLine {
+  		writeComment("", emptyArr, g)
```

Comment:
Could we use `nil` instead of an empty slice? Typed `nil` variables will take the empty value of their type, in this case, an empty string slice. I think `writeComment` would work still. The `emptyArr` is just kind of confusing imo

Test: https://play.golang.org/p/n7GluDmGICm